src/genie/libs/parser/iosxe/tests/ShowAvcSdServiceInfoSummary/cli/equal/golden_output_2_expected.py

```diff
@@ -0,0 +1,5 @@
+expected_output = {
+    "active_controller": {"status": "not configured"},
```

Comment:
if `Controller address isn't configured`, we don't need to create the level `active_controller`

Reply:
Updated